buildconfigs/oak_baremetal_app_crosvm.toml

```diff
@@ -0,0 +1,5 @@
+# This is the static build config that we use with the transparent release tool for building the provenance.
+# To be able to use it with the build tool from Transparent Release, additional information must be added.
+repo = "https://github.com/project-oak/oak"
+command = ["./scripts/xtask", "build-baremetal-variants"]
```

Comment:
Perhaps add a comment to clarify that we only generate proevenance for one variant even though this command builds both.

Reply:
Thanks. Done. But I think it is probably better to change the command to take an input parameter to choose the variant. That would avoid building both variants twice when we add the build config for the `qemu` variant too. I can fix this in a future PR.